ui/src/minard/utils/useMousePos.ts

```diff
@@ -1,18 +1,16 @@
-import {useState, useEffect, MutableRefObject} from 'react'
+import {useState, useEffect} from 'react'
 
-export const useMousePos = (
-  ref: MutableRefObject<Element>
-): [number, number] => {
+export const useMousePos = (el: Element): [number, number] => {
```

Comment:
Small nit. I would prefer this to return an object, simply because an array of two numbers doesn't convey much information to the user of the function.

Reply:
seems good, I've updated it